.travis.yml

```diff
@@ -105,7 +105,7 @@ jobs:
         - zip -qr code.zip build src package.json node_modules/
         - cd $TRAVIS_BUILD_DIR/serverless/unsubscribe-digest
         - npm install && npm run build
-        - zip -qr code.zip build package.json node_modules/
+        - zip -qr code.zip build src package.json node_modules/
```

Comment:
what did I miss here? Why do we zip src?

Reply:
This is so that the ts code can be viewed on Sentry